config/releases.exs

```diff
@@ -28,3 +28,8 @@ config :state_mediator, Realtime,
 config :state_mediator, State.Prediction, source: System.get_env("MBTA_TRIP_SOURCE")
 
 config :state_mediator, State.Vehicle, source: System.get_env("MBTA_VEHICLE_SOURCE")
+
+config :api_secrets,
```

Comment:
If you look in the deleted `secrets_manager.ex` file, you should see how the values should be configured. I think:

```ex
config :api_web, signing_salt: value
config : api_web, ApiWeb.Endpoint, secret_key_base: value
config :state_mediator, :commuter_rail_crowding, firebase_credentials: value
```

where `value` is the `System.fetch_env!`.

You can try deploying your branch to api-dev-green to see if it works.